webrtc/videoroom/lib/videoroom_web/templates/room/index.html.eex

```diff
@@ -2,8 +2,42 @@
     <a href="/">
         <h2>Membrane WebRTC video room demo</h2>
     </a>
-    <h3>Room: <%= @room_id %></h3>
-    <div id="videochat" class="VideoChat"> </div>
-    <div id="videochat-error" class="VideoChatError"> </div>
+    <h3 class="roomName">Room: <%= @room_id %></h3>
+    <div id="videochat" class="VideoChat">
+        <div id="videochat-error" class="VideoChatError"> </div>
+        <div id="videos-grid" class="grid-1"></div>
+    </div>
+
     <script src="<%= static_path(@conn, "/js/room.js") %>"></script>
 </div>
+<div id="media-controls", class="MediaControls">
+    <div id="video-on" class="ControlIcon" style="display: none;">
+        <svg fill="none" height="24" stroke="#fff" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg">
+            <polygon points="23 7 16 12 23 17 23 7"/>
+            <rect height="14" rx="2" ry="2" width="15" x="1" y="5"/>
+        </svg>
+    </div>
+    <div id="video-off" class="ControlIcon" style="display: none;">
+        <svg fill="none" height="24" stroke="#fff" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg">
+            <path d="M16 16v1a2 2 0 0 1-2 2H3a2 2 0 0 1-2-2V7a2 2 0 0 1 2-2h2m5.66 0H14a2 2 0 0 1 2 2v3.34l1 1L23 7v10"/>
+            <line x1="1" x2="23" y1="1" y2="23"/>
+        </svg>
+    </div>
+    <div id="mic-on" class="ControlIcon" style="display: none;">
+        <svg fill="none" height="24" stroke="#fff" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg">
+            <path d="M12 1a3 3 0 0 0-3 3v8a3 3 0 0 0 6 0V4a3 3 0 0 0-3-3z"/>
+            <path d="M19 10v2a7 7 0 0 1-14 0v-2"/>
+            <line x1="12" x2="12" y1="19" y2="23"/>
+            <line x1="8" x2="16" y1="23" y2="23"/>
+        </svg>
+    </div>
+    <div id="mic-off" class="ControlIcon" style="display: none;">
+        <svg fill="none" height="24" stroke="#fff" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg">
+            <line x1="1" x2="23" y1="1" y2="23"/>
+            <path d="M9 9v3a3 3 0 0 0 5.12 2.12M15 9.34V4a3 3 0 0 0-5.94-.6"/>
+            <path d="M17 16.95A7 7 0 0 1 5 12v-2m14 0v2a7 7 0 0 1-.11 1.23"/>
+            <line x1="12" x2="12" y1="19" y2="23"/>
+            <line x1="8" x2="16" y1="23" y2="23"/>
+        </svg>
```

Comment:
why are the SVGs inlined?

Reply:
I couldn't get it to work from files, fixed now.